generators/python-v2/codegen/src/ast/__test__/TypeInstantiation.test.ts

```diff
@@ -67,12 +68,24 @@ describe("TypeInstantiation", () => {
         expect(await writer.toStringFormatted()).toMatchSnapshot();
     });
 
-    it("dict", async () => {
-        TypeInstantiation.dict([
-            { key: TypeInstantiation.str("one"), value: TypeInstantiation.int(1) },
-            { key: TypeInstantiation.str("two"), value: TypeInstantiation.bool(true) }
-        ]).write(writer);
-        expect(await writer.toStringFormatted()).toMatchSnapshot();
+    describe("dict", () => {
```

Comment:
Adds some tests to make sure that references within python primitives get picked up.